language/move-vm/types/src/natives/function.rs

```diff
@@ -102,6 +114,24 @@ pub fn native_gas(table: &CostTable, key: NativeCostIndex, size: usize) -> GasUn
 #[macro_export]
 macro_rules! pop_arg {
     ($arguments:ident, $t:ty) => {{
-        $arguments.pop_back().unwrap().value_as::<$t>()?
+        match $arguments.pop_back().map(|v| v.value_as::<$t>()) {
+            None => {
+                return NativeResult::invariant_violation(vm::errors::PartialVMError::new(
```

Comment:
@tzakian How do I set this up properly so that it always refers to the correct crate?